test/README.md

```diff
@@ -75,8 +75,8 @@ The recommended way to do this yourself is to use python's `virtualenv`
 You can set up virtualenv for all these environments:
 
 ```console
-virtualenv -p python3.8 venv3.8
-virtualenv -p python3.9 venv3.9
+virtualenv -p python3.12 venv3.8
+virtualenv -p python3.12 venv3.9
```

Comment:
I think the idea was to show how to create different environments for different Python versions, so we should preserve that (and keep correct naming)
```suggestion
virtualenv -p python3.11 venv3.11
virtualenv -p python3.12 venv3.12
```

At the same time I don't even see Python 3.12 in the list of supported versions on PyPI. What's the policy these days, @terriko?

Reply:
The next version should support 3.9-3.12 but not yet supporting 3.13 (see #4663 ; it should be coming soon-ish).   I think 3.4 was supposed to support 3.12 but I may have missed updating the setup file before building the package, so we'll turn it on for 3.4.1.